compiler/passes/errorHandling.cpp

```diff
@@ -194,16 +292,18 @@ CallExpr* ErrorHandlingVisitor::haltExpr() {
   return new CallExpr(PRIM_RT_ERROR, new_CStringSymbol("uncaught error"));
 }
 
-// TODO: dtObject should be dtError, but we don't have that right now
 void lowerErrorHandling() {
+  INT_ASSERT(dtError->inTree());
+  INT_ASSERT(dtError->getRefType());
```

Comment:
I suggest removing the getRefType() assertion. In the future 'ref' formals will not require it. In the present, let the code fail instead at the point where the ref type is asked for.